src/DotNet.Status.Web/Controllers/GitHubHookController.cs

```diff
@@ -232,15 +241,20 @@ public class IssuesHookRepository
     {
         public string Name { get; set; }
         public IssuesHookUser Owner { get; set; }
+        public long Id { get; set; }
     }
 
     public class IssuesHookIssue
     {
         public int Number { get; set; }
         public string Title { get; set; }
+        public string Body { get; set; }
         public IssuesHookUser Assignee { get; set; }
         public ImmutableArray<IssuesHookLabel> Labels { get; set; }
         public ItemState State { get; set; }
+        public string Url { get; set; }
+        [JsonPropertyName("html_url")]
+        public string Html_Url { get; set; }
```

Comment:
Shouldn't it be named `HtmlUrl` or is there some reason why we need underscore here?

Reply:
I am not happy about it, but it was only way how I have found how to make this class to work with  both system.text.json and newtonsoft

Reply:
That's odd.. Both of these have attributes to define the JSON property name